packages/automated-dispute/tests/services/eboActor.spec.ts

```diff
@@ -489,42 +489,30 @@ describe("EboActor", () => {
     });
 
     describe("settleDispute", () => {
-        it("escalates dispute when BondEscalationModule_ShouldBeEscalated error occurs", async () => {
+        it("escalates dispute when amountOfPledgesForDispute <= amountOfPledgesAgainstDispute", async () => {
```

Comment:
Remember to change this test and the next one too:
```suggestion
        it("escalates dispute when amountOfPledgesForDispute === amountOfPledgesAgainstDispute", async () => {
```